scripts/setup-proof.py

```diff
@@ -22,7 +22,7 @@ def read_proof_template(filename):
 
 def write_proof_template(lines, filename, directory):
     with open(os.path.join(directory, filename), "w") as data:
-        data.write('\n'.join(lines))
+        data.write('\n'.join(lines)+'\n')
```

Comment:
Why not just
```python
print('\n'.join(lines), file=data)
```
print appends a newline.

Reply:
I followed Saswat's recommendation to use writelines().